app/views/admin/suppliers/delivery_partners/_lead_providers_cohorts.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= f.govuk_check_boxes_fieldset(
+  :lead_providers,
+  legend: { text: "Which lead providers and cohorts are #{@delivery_partner_form.name} working with?", size: "m" },
+  hint: -> { tag.p do
+    concat(tag.span "If your lead provider isn't here you need to ")
+    concat(govuk_link_to("add them", new_admin_supplier_path))
+    concat(".")
+  end
+  }
+) do %>
+  <% @delivery_partner_form.available_lead_providers.each do |lead_provider| %>
+    <%= f.govuk_check_box :lead_providers, lead_provider.id, label: { text: lead_provider.name }, link_errors: true do %>
+      <%= f.govuk_check_boxes_fieldset(
+        :provider_relationship_hashes,
+        legend: { hidden: true },
+        hint: { text: "What cohort/academic year are they working with?" },
+        ) do %>
+        <% lead_provider.cohorts.each do |cohort| %>
+          <%= f.govuk_check_box(
+                :provider_relationship_hashes,
+                DeliveryPartnerForm.provider_relationship_value(lead_provider, cohort),
+                label: { text: cohort.display_name },
+                link_errors: true)
+          %>
+        <% end %>
```

Comment:
It's also used in the controller

Reply:
Naturally what I'm suggesting higher up would lead to 'hmm, should we use value objects to store our form data, rather than Rails models?', and maybe the answer is 'not yet, we are happy with attribute accessors on the models', or maybe yes, lets have a separate PR to add that pattern here'.

Reply:
I think that makes sense, I'm generally in favour of moving logic out of controllers and especially views. I don't think I'll have time to do that here, but feel free to open a ticket for it (or a PR 😛 )

Reply:
Anything you want changed for this in this PR @kicferk1 ?

Reply:
A `todo` comment with link to https://trello.com/c/ursP6wtZ/328-refactor-how-we-present-partnerships-when-creating-updating-them would be nice. Either here or in the form itself.